service/src/test/java/bio/terra/cbas/dao/TestMethodDao.java

```diff
@@ -50,22 +58,40 @@ class TestMethodDao {
           methodSource,
           workspaceId);
 
+  @Container
+  static JdbcDatabaseContainer postgres =
+      new PostgreSQLContainer("postgres:14")
+          .withDatabaseName("test_db")
+          .withUsername("test_user")
+          .withPassword("test_password");
+
+  @DynamicPropertySource
+  static void postgresProperties(DynamicPropertyRegistry registry) {
+    registry.add("spring.datasource.jdbc-url", postgres::getJdbcUrl);
+    registry.add("spring.datasource.username", postgres::getUsername);
+    registry.add("spring.datasource.password", postgres::getPassword);
+  }
+
   @BeforeAll
-  void setUp() {
+  static void setup() {
+    postgres.start();
+  }
+
+  @BeforeEach
+  void init() {
     int recordsCreated1 = methodDao.createMethod(method1);
     int recordsCreated2 = methodDao.createMethod(method2);
 
     assertEquals(1, recordsCreated1);
     assertEquals(1, recordsCreated2);
   }
 
-  @AfterAll
-  void cleanUp() {
-    int recordsDeleted1 = methodDao.deleteMethod(methodId1);
-    int recordsDeleted2 = methodDao.deleteMethod(methodId2);
-
-    assertEquals(1, recordsDeleted1);
-    assertEquals(1, recordsDeleted2);
+  @AfterEach
+  void cleanupDb() throws SQLException {
+    DriverManager.getConnection(
+            postgres.getJdbcUrl(), postgres.getUsername(), postgres.getPassword())
+        .createStatement()
+        .execute("DELETE FROM run_set; DELETE FROM method_version; DELETE FROM method;");
```

Comment:
make sure to delete github method details and any other tables that I've missed here